sql/views.sql

```diff
@@ -1,25 +1,3 @@
 /*
```

Comment:
Is this file still required? I looked up `EDH_WMS_STAGING.DATABASECHANGELOG` in Develop and `sql/views.sql` is registered under differed MD5 Hash. This file is going to fail Liquibase once run in Develop.

Reply:
Ah yes, I can't touch the file without Liquibase going mad.  I tried all sorts but it wouldn't work for both a fresh install and an upgrade.  I've reverted the whole file.  On a fresh install, it creates the old view definition and then replaces it with the new one.